codespell_lib/data/dictionary.txt

```diff
@@ -7795,11 +7880,27 @@ bridg->bridge
 brievely->briefly
 brievety->brevity
 brigde->bridge
+brigdes->bridges
 brige->bridge
 briges->bridges
+brigh->bright
```

Comment:
Perhaps `brig`? Keys **G** and **H** are neighbours.

Reply:
I couldn't find this.